### ISSUE TITLE
Help for DeepGlobe dataset

### ISSUE THREAD

**Issue by Lelouch-Ice** [TAMU-VITA/GLNet#2]
Hello, good work. I am trying to run your code on the DeepGlobe dataset, but I have difficulty in getting the DeepGlobe dataset  as the websites didn't respond to my request for two weeks.  Is it convenient for your to provide a copy of DeepGlobe dataset on val set and test set? My personal email is bingwangnpu@163.com. 
Besides, I found there is some gap in the evaluation results in your paper and benchmark results in the official [website](https://competitions.codalab.org/competitions/18468#results), is there some difference on your evaluation methods?

**Comment by chenwydj** [comment]
Hi! Thank you for your interest!
1. Sorry that following the challenge policy I cannot share the dataset to others.
2. We report our accuracy on the Deep Globe validation dataset, since the challenge is already closed. We use the exact same metric code in this repo for the evaluation in our paper. I think there may be some domain shift between the validation and test set.

**Comment by nidetaoge** [comment]
Wuyang, congratulations on your work~

**Comment by Lelouch-Ice** [comment]
> Hi! Thank you for your interest!
> 
> 1. Sorry that following the challenge policy I cannot share the dataset to others.
> 2. We report our accuracy on the Deep Globe validation dataset, since the challenge is already closed. We use the exact same metric code in this repo for the evaluation in our paper. I think there may be some domain shift between the validation and test set.

Hello, after  waitting for nearly a month, I was accepted to access the dataset. As you mentioned you use the same metric code for evaluation on val set, but I didn't found label fliles corresponding to the val set,  yet your code`dataset_val = DeepGlobe(os.path.join(data_path, "crossvali"), ids_val, label=True)`, how can you evaluate val set without corresponding labels?

**Comment by lamhoangtung** [comment]
Hi @chenwydj . Thanks for your great work. I trying to recreate your result for DeepGlobe datasets.
I'm facing the same issue as @Lelouch-Ice where I can't find any label files corresponding to the val set, and the test set, yet your code:

https://github.com/chenwydj/ultra_high_resolution_segmentation/blob/e3ad351d2291ec03b6108dd17b9be57fa89ba4ed/train_deep_globe.py#L53-L56

How can you evaluate val set without corresponding labels ? Where did that `crossvali` and `offical_crossvali` folder come from ? Also the path for training data is also different from your code ...

Did you do something to the dataset before using it ? If yes, what exactly are those step ?

Looking forward to your answers. Thanks a lot

**Comment by dexception** [comment]
Can anyone share the a pretrained model so that others can just run and see if it is worth the effort ?

**Comment by chenwydj** [comment]
Hi! We updated the instructions for training and evaluation in the Readme, and also uploaded the pretrained model for the Deep Globe. Please take a look and give a try! :)
